<!doctype html>
<html lang="en">
<head>
    <meta charset="utf-8">
    <title>Lyla Atta</title>
    <link rel="stylesheet" href="styles_cv.css">
</head>

<body>
	<div id="upperborder">
		hello
	</div>

	<div id="top">
		<h1><a class="home" href="../index.html">Lyla Atta </a><span style="font-size: 42px;font-weight:normal; font-color: #c2c2c2">> CV</span></h1>
    	<h4>MD-PhD Candidate</h4>
    	<h5>Johns Hopkins University</h5>

        <div class="topnav">
            <a class="bar" href="#research">Research</a>
            <a class="bar" href="#teaching">Teaching</a>
            <a class="bar" href="#projects">Projects</a>
            <a class="bar" href="#mentorship">Mentorship</a>
            <a class="bar" href="#professional">Professional Service</a>
        </div>
        <div class="topnav" style="padding-top: 0px; padding-bottom: 10px;">
            <a class="bar" href="#pubs">Publications</a>
            <a class="bar" href="#presentations">Presentations and Posters</a>
            <a class="bar" href="#education">Education</a>
            <a class="bar" href="https://github.com/LylaAtta/LylaAtta.github.io/raw/main/LylaAtta_CV_20231204.pdf">cv.pdf</a>
        </div>

    </div>

    <div id="body">

    <h6><a id="research"></a>Research Experience</h6>
    <ul class="main">
        <li class="main"><b>Graduate Student</b> 
            <a class="general" href="https://jef.works/">JEFworks lab</a>, BME, JHU<br> 
            <em>August 2020 - Present.</em><br>
            <b>Developing statistical and computational tools for the analysis of spatially resolved transcriptomics data.</b></li>
            <ul class="sub">
                <li class="sub">Gene count normalization in single-cell imaging-based spatially resolved transcriptomics. <a class="general" href="https://www.biorxiv.org/content/10.1101/2023.08.30.555624v1"><em>bioRxiv</em> 2023 <em>under review</em></a>.</li>
                <li class="sub">VeloViz: RNA-velocity informed embeddings for visualizing cellular trajectories. <a class="general" href="https://doi.org/10.1093/bioinformatics/btab653"><em>Bioinformatics 2021</em></a>. <a class="general" href="https://jef.works/veloviz/">Software and tutorials</a>.</li>
                <li class="sub">Computational challenges and opportunities in spatially resolved transcriptomic data analysis. <a class="general" href="https://www.nature.com/articles/s41467-021-25557-9"><em>Nature Communications 2021</em></a>.</li>
                <li class="sub">Character- izing cell-type spatial relationships across length scales in spatially resolved omics data. <em>bioRxiv, 2023, under review.</em></a>.</li>
                <li class="sub">Alignment of spatial transcrip- tomics data using diffeomorphic metric mapping. <em>Nature Communications, 2023, accepted.</em></a>.</li>
                <li class="sub">Soluble PD-L1 reprograms blood monocytes to prevent cerebral edema and facilitate recovery after ischemic stroke. <em>Brain Behavior and Immunity, 2023. accepted. </em></a>.</li>
                <li class="sub">Reference-free cell-type deconvolution of pixel-resolution spatially resolved tran- scriptomics data. <a class="general" href="https://www.nature.com/articles/s41467-022-30033-z"><em>Nature Communications, 2022.</em></a>.</li>
                <li class="sub">Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma. <a class="general" href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00268-3"><em>Cancer Cell 2021</em></a>.</li>
            </ul>
        <li class="main"><b>Graduate Rotation Student</b>
            Elana Fertig Lab, JHU<br> 
            <em>July 2020 - August 2019.</em><br>
            <b>Statistical models to simulate immune cell single-cell RNA sequencing.</b></li>
        <li class="main"><b>Research Associate</b>
            Scott Manalis Lab, MIT<br> 
            <em>June 2017 - June 2018.</em><br>
            <b>Investigated single cell biophysical measurements and gene expression as markers of immune checkpoint inhibitor therapy in melanoma.</b></li>
        <li class="main"><b>Undergraduate Researcher</b>
            Douglas Lauffenburger Lab, MIT<br> 
            <em>March 2017 - June 2017.</em><br>
            <b>Investigated signaling networks underlying resistance to targeted cancer therapy.</b></li>
            <ul class="sub">
                <li class="sub">Tested dose dependent effects of drug combinations on Triple Negative Breast Cancer (TNBC) cell lines.</li>
                <li class="sub">Developed mathematical growth model comparing TNBC cell line drug response.</li>
                <li class="sub">Analyzed bulk gene expression to identify mediators of non-genetic resistance to EGFR pathway inhibitors.</li>
            </ul>
        <li class="main"><b>Data Science Coursework</b>
            Statistics and Data Science, MIT<br> 
            <em>Fall 2016.</em><br>
            <b>Network Analysis of Phosphoproteomic Data in Glioblastoma.</b></li>
            <ul class="sub">
                <li class="sub">Created graphical models of protein signaling interactions based on receptor tyrosine kinase phosphorylation data from GBM mouse models.</li>
                <li class="sub">Implemented graphical network analysis approaches to identify differences between tumor and control network models.</li>
                <li class="sub">Identified significant changes in signaling networks between tumor and control and identified cytoskeletal proteins marking these changes.</li>
            </ul>
        <li class="main"><b>Undergraduate Researcher</b>
            Ron Weiss Lab, MIT<br> 
            <em>November 2014 - March 2016.</em><br>
            <b>Synthetic mammalian circuit for <em>in vivo</em>sensing and response to extracellular biomarkers.</b></li>
            <ul class="sub">
                <li class="sub">Automated, distribution-based clustering of flow cytometry data in Matlab.</li>
                <li class="sub">Designed an antibody-based extracellular platform for in vivo detection of biomarkers and transduction of an intracellular response.</li>
                <li class="sub">Collaborated to build detection platform using recombinant gene cloning techniques and tested functionality in Human Embryonic Kidney cells using flow cytometry and confocal microscopy.</li>
            </ul>
    </ul>

<hr>

    <h6><a id="teaching"></a>Teaching Experience</h6>
    <ul class="main">
        <li class="main"><b>Genomic Data Visualization</b><br>
            5 Students, Biomedical Engineering, JHU.<br> 
            <em>Spring 2022.</em><br>
            <b>Developed and taught guest lecture, graded homework and exams, provided feed- back for improvement.</b></li>
        <li class="main"><b>Critical Appraisal of Clinical Studies</b><br>
            15 Students, School of Medicine, JHU.<br> 
            <em>Fall 2020.</em><br>
            <b>Developed and taught lesson plan, led student discussion for first-year medical school selective focused on critical appraisal of COVID-19 therapeutic trials.</b></li>
        <li class="main"><b>Analysis of Biomolecular and Cellular Systems (20.320)</b><br>
            40 Students, Biological Engineering, MIT.<br> 
            <em>Fall 2016.</em><br>
            <b>Developed and taught lesson plans, created supplementary course notes, problem set and exam solutions for undergraduate core biological engineering course focused on mechanistic modeling of biomolecular reactions using differential equations in Matlab.</b></li>
    </ul>

<hr>

    <h6><a class="anchor" id="projects"></a>Activities and Projects</h6>
    <ul class="main">
        <li class="main"><b>Open Case Studies</b><br>
            JHU School of Medicine<br> 
            <em>November 2021 - March 2022</em><br>
            <b>Contributed to Educators Guide for using Open Case Studies for data science instruction.</b></li>
        <li class="main"><b>JHU Medical School Admissions Committee</b><br>
            JHU School of Medicine<br> 
            <em>August 2021 - March 2022</em><br>
            <b>Interviewed 31 JHUSOM applicants, reviewed and summarized applications, and provided admission input to faculty admissions committee.</b></li>
        <li class="main"><b><a class="general" href="https://www.panfab.org/">Greater Boston Pandemic Fabrication Team</a></b><br>
            <em>March 202 - September 2021</em><br>
            <b>Co-led design and testing of Powered Air-Purifying Respirators to address COVID-19 PPE shortages. (Kothakonda <em>et. al.</em></b></li>
        <li class="main"><b>Center for RISC</b><br>
            University of Chicago<br> 
            <em>July 2020 - September 2020</em><br>
            <b>Adapted data science curriculum for use by students with limited programming experience.</b></li>
        <li class="main"><b>Independent Student Analysis Committee</b><br>
            JHU School of Medicine<br> 
            <em>March 2020</em><br>
            <b>Analyzed student survey to identify strengths and areas of improvement in medical school experience for LCME re-accreditation process.<br>
            Recommended strategies to improve pre-clinical curriculum, diversity and inclusion, student mistreatmet based on student survey results.</b></li>
        <li class="main"><b>Student Curriculum Review Team</b><br>
            JHU School of Medicine<br> 
            <em>August 2018 - March 2020</em><br>
            <b>Analyzed evaluations for medical school pre-clinical courses to identify strengths and weaknesses.<br>
            Organized student town hall to solicit student feedback to improve pre-clinical Genetics and Metabolism courses.<br>
            Designed and analyzed a student survey assessing effectiveness of the Student Curriculum Review Team in facilitating curricular improvements. (Kumar <em>et. al.</em>)<br></b></li>
        <li class="main"><b>Medical School Peer Advising Leader</b><br>
            JHU School of Medicine<br> 
            <em>March 2019 - March 2020</em><br>
            <b>Coordinated matching of 120 first-year medical school students to second-year mentors.<br>
            Organized social and educational events facilitating exhange of advice through multiple transitions in medical school pre-clinical curriculum.</b><br></li>
        <li class="main"><b>Qualified Bilingual Staff Speaker</b><br>
            Johns Hopkins Medicine<br> 
            <em>March 2020 - present</em><br>
            <b>Qualified to communicate in Arabic with patients within the scope of clinical student role.<br></b></li>
        <li class="main"><b>Undergraduate Board President</b><br>
            Biological Engineering, MIT.<br> 
            <em>February 2016 - March 2017</em><br>
            <b>Established first annual BE Research Symposium for students to present their research.<br>
            <b>Planned, administered, and analyzed the first feedback survey and forum soliciting feedback from graduating seniors and presented results to faculty committee.<br>
            <b>Led a weeklong Freshman Pre-orientation Program for students interested in biological engineering.<br>
            <b>Created a sophomore orientation program for students starting the Biological Engineering major.<br>
            </b></li>
    </ul>

<hr>

    <h6><a id="mentorship"></a>Mentorship</h6>
    <ul class="main" style="font-weight: normal; margin-top: 0px; margin-bottom: 0px">
        <li class="main" style="margin-top: 0px; margin-bottom: 0px;"><b>JEFworks Lab</b><br>
            <em>2020 - 2021</em><br>
            1 undergraduate and 1 high school student<br> 
            </li>
        <li class="main"><b>JHU School of Medicine</b><br>
            Peer Advisor, 2 first-year medical students<br> 
            <em>2019 - 2020</em><br></li>
        <li class="main"><b>MIT Biological Engineering</b><br>
            Associate Advisor, 4 sophomore students<br> 
            <em>2019 - 2020</em><br></li>
        <li class="main"><b>MIT Saturday Engineering Enrichment and Discovery Academy</b><br>
            Mentor, ~15 students<br> 
            <em>2016</em><br></li>
        <li class="main"><b>MIT DynaMIT STEM summer program</b><br>
            Mentor, 4 students<br> 
            <em>2016</em><br></li>
    </ul>

<hr>

    <h6><a id="professional"></a>Professional Service</h6>
    <ul class="main">
        <li class="main"><b>Peer Review:</b> </li>
            <ul class="sub" style="font-weight: normal;">
                <li class="sub">Bioinformatics,<em>2021.</em></li>
                <li class="sub">Genome Research,<em>2021.</em></li>
                <li class="sub">BMJ Open,<em>2021.</em></li>
            </ul>  
    </ul>


<hr>


    <h6><a id="pubs"></a>Publications</h6>
    <ul class="main">
        <li class="main"><b>Graduate</b> 
            <ul class="sub">
                <li class="sub"><b><u>Atta L.*</u></b>, Sahoo A., Fan J., VeloViz: RNA-velocity informed embeddings for visualizing cellular trajectories. <a class="general" href="https://doi.org/10.1093/bioinformatics/btab653"><em>Bioinformatics, 2021</em></a>. <a class="general" href="https://jef.works/veloviz/">Software and tutorials</a>.</li>
                <li class="sub"><b><u>Atta L.*</u></b>, Fan J., Computational challenges and opportunities in spatially resolved transcriptomic data analysis. <a class="general" href="https://www.nature.com/articles/s41467-021-25557-9"><em>Nature Communications 2021</em></a>.</li>
                <li class="sub">Hara T., Chanoch-Myers R., Mathewson N.D., Myskiw C., <b><u>Atta L.</u></b>, Bussema L., Eichhorn S.W., Greenwald A.C., Kinker G.S., Rodman C., Gonzalez Castro L.N., Wakimoto H., Rozenblatt-Rosen O., Zhuang X., Fan J., Hunter T., Verma I.M., Wucherpfennig K.W., Regev A., Suvà M.L., Tirosh I., Interactions between cancer cells and immune cells drive transitions to mesenchymal-like states in glioblastoma. <a class="general" href="https://www.cell.com/cancer-cell/fulltext/S1535-6108(21)00268-3"><em>Cancer Cell 2021</em></a>.</li>
                <li class="sub">Miller B., Huang F., <b><u>Atta L.</u></b>, Sahoo A., Fan J., Reference-free cell-type deconvolution of pixel-resolution spatially resolved transcriptomics data. <a class="general" href="https://www.nature.com/articles/s41467-022-30033-z"><em>Nature Communications, 2022</em></a>.</li>
            </ul>
        <li class="main"><b>Addressing COVID-19 PPE shortages</b>
            <ul class="sub">
                <li class="sub">Kothakonda A.*, <b><u>Atta L.*</u></b>, Plana D.*, Ward F.*, Davis C., Cramer A., Moran R., Freake J., Tian E., Mazor O., Gorelik P., Van C., Hansen C., Yang H., Li Y., Sinha M.S., Li J., Yu S.H., LeBoeuf N.R., Linville-Engler B., Sorger P.K. De Novo Powered Air-Purifying Respirator Design and Fabrication for Pandemic Response. <a class="general" href="https://www.frontiersin.org/articles/10.3389/fbioe.2021.690905/full"><em>Front. bioeng. biotechnol., 2021.</em></a>.</li>
                <li class="sub">Antonini M.J., Plana D., Srinivasan S., <b><u>Atta L.</u></b>, Achanta A., Yang H., Cramer A., Freake J., Sinha M.S., Yu S.H., LeBoeuf N.R., Linville-Engler B., Sorger P.K.A Crisis-Response Framework for Medical Device Development During the COVID-19 Pandemic. Frontiers in Digital Health., 2021. <a class="general" href="https://www.frontiersin.org/articles/10.3389/fdgth.2021.617106/full"><em>Frontiers in Digital Health., 2021.</em></a>.</li>
                <li class="sub">Tyan K., Levin A., Avalos-Pacheco A., Plana D., Rand E.A., Yang H., Maliszewski L.E., Chylek L.A., <b><u>Atta L.</u></b>, Tye M.A., Carmack M.M., Oglesby N.S., Burgin S., Yu S.H., LeBoeuf N.R., Kemp J.M. Considerations for the Selection and Use of Disinfectants Against SARS-CoV-2 in a Healthcare Setting. <a class="general" href="https://academic.oup.com/ofid/article/7/9/ofaa396/5899802"><em>Open Forum for Infectious Disease, 2020.</em></a>.</li>
            </ul>
        <li class="main"><b>Medical Education</b>
            <ul class="sub">
                <li class="sub">Kumar P., Pickering C.M., <b><u>Atta L.</u></b>, Burns A.G., Chu R.F., Gracie T., Qin C.X., Whang K.A., Goldberg H.R., Student Curriculum Review Team, 8 Years Later. <a class="general" href="https://www.tandfonline.com/doi/abs/10.1080/0142159X.2020.1841891"><em>Medical Teacher, 2021.</em></a></li>
            </ul>
        <li class="main"><b>Undergraduate</b>
            <ul class="sub">
                <li class="sub">Claas A.M., <b><u>Atta L.</u></b>, Gordonov S., Meyer A.S., Lauffenburger D.A. Systems Modeling Identifies Divergent Receptor Tyrosine Kinase Reprogramming to MAPK Pathway Inhibition. <a class="general" href="https://link.springer.com/article/10.1007/s12195-018-0542-y"><em>Cell. Mol. Bioeng., 2018.</em></a>.</li>
            </ul>
    </ul>

<hr>

    <h6><a id="presentations"></a>Presentations and Posters</h6>
    <ul class="main" style="font-weight: normal;">
        <li class="main">VeloViz: RNA velocity-informed 2D embeddings for visualizing cellular trajectories. Nature Bioengineering Solutions for Biology and Medicine July 2022. <em>Flash-talk and Poster.</em> <br><b>Best Poster Award.</b></li>
        <li class="main">Understanding transcriptional cell state changes within spatial contexts. NHGRI Genome TDCC Technology Area Forum May 2022. <em>Invited Talk.</em></li>
        <li class="main">VeloViz: RNA velocity-informed 2D embeddings for visualizing cellular trajectories. Neural Information Processing Systems Conference December 2021. <em>Invited Talk.</em></li>
        <li class="main">VeloViz: RNA velocity-informed 2D embeddings for visualizing cellular trajectories. JHU Bioinformatics Symposium October 2021. <em>Flash-talk and Poster.</em></li>
        <li class="main"><a class="general" href="https://www.youtube.com/watch?v=JePjSpzI-2o">VeloViz: RNA velocity-informed 2D embeddings for visualizing cellular trajectories.</a> Cold Spring Harbor Labs Single-Cell Meeting November 2021. <em>Flash-talk and Poster.</em></li>
        <li class="main">VeloViz:RNA velocity-informed 2D embeddings for visualizing cellular trajectories. Broad Institute Single-Cell Working Group April 2021. <em>Invited Talk.</em></li>
        <li class="main">VeloViz: RNA velocity-informed 2D embeddings for visualizing cellular trajectories. Women of Whiting Symposium, Johns Hopkins University, April 2021. <em>Poster.</em></li>
        <li class="main"><a class="general" href="https://hmcrs.org/events/pandemic-response/">De novo Powered Air-Purifying Respirator Design and Manufacture to Address Commercial Supply Shortages.</a> Pandemic Response: Resilience and Recovery in the Era of COVID-19, Harvard-MIT Center for Regulatory Science, June 2020. <em>Invited Talk.</em></li>
        <li class="main">Prediction of BET Bromodomain Inhibitor Sensitization to Targeted Cancer Drugs. Znaty-Merck Prize for Research in BE, May 2017. <em>Presentation.</em></li>
        <li class="main">Using Native Transcriptomics to Predict RTK Response to Targeted Cancer Drugs. Undergraduate BE Research Symposium, May 2017. <em>Presentation.</em>.</li>
        <li class="main">Transcriptome Changes as Mediators of Intrinsic Cancer Drug Resistance. MIT Amgen Scholars Program, Aug. 2016. <em>Poster.</em></li>
        <li class="main">B-Cell Receptor-Based Genetic Sensor for Extracellular Biomarker Detection. MIT Center for Biomedical Innovations Cell and Gene Therapy Biomanufacturing Summit, Nov. 2015. <em>Poster.</em></li>
        <li class="main">Beta-amyloid Detection for Alzheimer’s Diagnosis and Therapy. Institute of Biological Engineering, Mar. 2015. <em>Presentation.</em></li>
        
    </ul>

<hr>

    <h6><a id="education"></a>Education</h6>
    <ul class="main">
        <li class="sub"><b>Johns Hopkins University</b> MD-PhD Candidate. <em> 2018 - Present.</em><li>
        <li class="sub"><b>Massachusetts Institute of Technology</b> Bachelors of Science,  Major: Biological Engineering, Minor: Statistics and Data Science. <em>2013 - 2017.</em><li>        
    </ul>

<hr>

	</div>
</body>
</html>
